numpy/random/_generator.pyx

```diff
@@ -3412,19 +3412,34 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized geometric distribution.
 
+        References
+        ----------
+
+        .. [1] Wikipedia, "Geometric distribution",
+               https://en.wikipedia.org/wiki/Geometric_distribution
+
         Examples
         --------
-        Draw ten thousand values from the geometric distribution,
-        with the probability of an individual success equal to 0.35:
+        Draw 10.000 values from the geometric distribution, with the 
+        probability of an individual success equal to ``p = 0.35``:
 
+        >>> p, size = 0.35, 10000
         >>> rng = np.random.default_rng()
-        >>> z = rng.geometric(p=0.35, size=10000)
+        >>> sample = rng.geometric(p=p, size=size)
 
-        How many trials succeeded after a single run?
+        What percentage of trials succeeded after a single run?
```

Comment:
If you want to use percentages you need to multiply the result by 100. Maybe proportion would be more appropriate?
```suggestion
        What proportion of trials succeeded after a single run?
```